src/Sarif.FunctionalTests/ConverterTestData/StaticDriverVerifier/cancelspinlock_bug1.tt.sarif

```diff
@@ -346,7 +346,7 @@
                       "nestingLevel": 2,
                       "importance": "unimportant",
                       "properties": {
-                        "currentDataValues": "^\u0026sdv_ControllerIrp=8520",
+                        "currentDataValues": "^&sdv_ControllerIrp=8520",
```

Comment:
Yay! What did the problem turn out to be?

Reply:
No idea, I didn't change anything...on purpose at least

Reply:
Newtonsoft version 10 vs 11, as I originally speculated?